### PR TITLE
Add rb-csvlint

### PR DESCRIPTION
**Pull request by svengiegerich**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by janjagusch** [comment]
> Before this package will be runnable specific ruby dependencies need to be added to conda-forge (as well):
> 
> * [ ]  `open_uri_redirections`
> * [ ]  `escape_utils`
> * [ ]  `rack`
> * [ ]  `net-http-persistent`
>   
>   * [ ]  `connection_pool`

@svengiegerich, i think all necessary packages have been released. :)

File: recipes/csvlint/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 886c87441af105065e620386434ae2cd9d9c69cfcd532335ae52f1cd15f419c8
+
+build:
+  noarch: generic
```
Comment (janjagusch) [inline recipes/csvlint/meta.yaml:13]:
```suggestion
```

**Comment by janjagusch** [comment]
@conda-forge-admin, please restart ci

File: recipes/csvlint/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - rb-open_uri_redirections
+    - rb-activesupport
+    - rb-addressable
+    - rb-typhoeus
+    - rb-escape_utils
+    - rb-uri_template
+    - rb-thor
+    - rb-rack
+    - rb-net-http-persistent
+    - rb-concurrent-ruby <1.1
```
Comment (janjagusch) [inline recipes/csvlint/meta.yaml:34]: The pin works for now, but it probably makes sense to pin the ruby version in the [`rb-concurrent-ruby` feedstock](https://github.com/conda-forge/rb-concurrent-ruby-feedstock/blob/main/recipe/meta.yaml).

**Review by janjagusch — APPROVED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**